chef-config/lib/chef-config/config.rb

```diff
@@ -33,6 +33,7 @@
 require "addressable/uri" unless defined?(Addressable::URI)
 require "openssl" unless defined?(OpenSSL)
 require "yaml"
+require "chef/dist"
```

Comment:
this gives us a circlular dep between chef-config and chef itself.

Reply:
good point; I'll add a dist.rb to chef-config to avoid that

Reply:
See latest commit for proposed fix, but this has me wondering if dist.rb shouldn't simply be entirely owned by chef-config. Some quick and dirty sed could address that if we want to try it.